src/components/Nav.js

```diff
@@ -1,8 +1,20 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
+import { LocationsContext } from '../context/LocationsContext';
 import './Nav.css';
 
 export default function Nav() {
+  const { allowLocation, setAllowLocation, loading } = useContext(
+    LocationsContext,
+  );
+
+  //Make this async and only toggle after response from api call is returned
+  const handleLocation = async () => {
+    console.log('location toggle!');
```

Comment:
Consider adding a `// TODO` comment here to remove this after we've done the next ticket where the user can turn off location sharing

Reply:
- [ x] Added the TODO:

Reply:
- [x] Added the TODO!

Reply:
Thank you!